README.md

```diff
@@ -188,3 +188,11 @@ Payload for this reponse (ContractAgreement) will be read from file, named contr
 you wish to send different ContractAgreement, just modify content of this file.
 This way, we are simulating simple contract negotiation sequence.
 
+# Description Request/Response Message
+
+When receiving a Description Request Message we are preparing a response by creating a Description Response Message for the header part and putting the whole Self Description(ids:BaseConnector) or requested element from Self Description (ids:Resource) in the payload.
+In both cases a GET request is sent to the ECC in order to fetch the Self Description. The following properties need to be configured and correspond the Self Description configuration from the ECC.
+```
+application.ecc.RESTprotocol=http
```

Comment:
What happens in ECC is in "https" mode? Does the port remain the same?

Reply:
That is mentioned in the application.properties, but I agree that it could be also stated here.